### DISCUSSION TITLE
Can knife4j support typescript preview and paste?

### DISCUSSION THREAD

**Discussion by geekftz** [xiaoymin/knife4j#495] [Ideas]
so many teams are going to use typescript to write web projects, I think if it support it will be good for developers, thks!

**Comment by xiaoymin** [comment]
Knife4j is open to adapting more languages or frameworks, but I have no  experience in TypeScript projects, so if you have more practical experience in TypeScript, you can contribute together

**Comment by xiaoymin** [comment]
Some developers have contributed PRs about TypeScript's code generator feature. Does it meet your requirements?

https://github.com/xiaoymin/swagger-bootstrap-ui/pull/489
